app/src/main/res/values/strings.xml

```diff
@@ -26,7 +26,10 @@
     <string name="action_ok">OK</string>
     <string name="action_cancel">Cancel</string>
     <string name="msg_delele_album">Do you want delete album ? </string>
+    <string name="msg_remove_track_album">Do you want remove track ? </string>
     <string name="action_add">Add</string>
+    <string name="action_remove">Remove</string>
+    <string name="action_add_favorite">Add Favorite</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.action_add_favorite` appears to be unused
 Unused resources make applications larger and slow down builds.